docs/user_guides/debugging/index.rst

```diff
@@ -18,7 +18,7 @@ See :ref:`debugging-features-and-limitations`.
 Requirements
 ------------
 
-`Intel® Distribution for GDB` is needed for `numba-dppy`'s debugging features to work.
+`Intel® Distribution for GDB` is required for `numba-dppy`'s debugging features to work.
```

Comment:
back-ticks around numba-dppy. Sorry again as I might have put them in the first place.

Reply:
It will be good to get some text from Matti to add here to explain why we need Intel GDB.

Reply:
we have explanation here https://intelpython.github.io/numba-dppy/latest/user_guides/debugging/index.html#requirements